README.md

```diff
@@ -58,7 +58,8 @@ expressOasGenerator.init(
   60 * 1000,
   'api-docs',
   ['User', 'Student'],
-  ['users', 'students']
+  ['users', 'students'],
+  ['production']
```

Comment:
Instead of having an array of environment strings to ignore, I'd consider having a boolean-returning function that should determine if the module should not be ran.

```suggestion
  () => process.env.NODE_ENV !== "production"
```

(because this would be more extensible I'd think).

but then, a user could themselves have an `if` statement to check this. So not sure if we can rely only on the `NODE_ENV` variable, but if so - perhaps that's enough and we don't need my suggestion.

Reply:
I like neither my solutions or yours LOL. @mpashkovskiy can suggest something else here 😄

Reply:
BTW why a boolean returning function and not a boolean alone? Maybe I could add `runMiddlewares` and `serveApiDocs` booleans. What do you think?

Reply:
The number of parameters in that method/option object scares me.

Shall we:
1. avoid adding an extra param here
2. in case `process.env.NODE_ENV === "production"` avoid collecting requests/responses and serving UI+spec 
3. in case `process.env.NODE_ENV` is undefined throw an exception/huge warning

What do you think?

Reply:
Not sure, undefined `process.env.NODE_ENV` is not recommended but valid. We shouldn't throw, maybe a warning (I prefer to implement a logging library for that, not a `console.warn`).

About the number of parameters: For the `options` object it shouldn't be a problem, as most of them are optionals and have default values. As you wish!

Reply:
Just included points 2 and 3. Will finish once we decide number 1 : 
* Ok the way it is (ignoreNodeEnvironments param)
* Include a boolean param 
* No params at all

Reply:
Let me know which solution works best for you guys @mpashkovskiy @kiprasmel

Reply:
How about havin boolean param with three stages:
* undefined - default - rely on `ignoreNodeEnvironments`
* true - serve spec
* false - don't serve the spec

Reply:
> Thank you guys for an effort! I'll test locally and publish new version.

Thanks @mpashkovskiy ! I'm including the boolean arg `alwaysServeDocs`. Would you revert the merged PR (or should we reopen) to ship it all together?